src/main/java/com/google/devtools/build/lib/remote/RemoteExecutionService.java

```diff
@@ -1078,6 +1100,15 @@ public InMemoryOutput downloadOutputs(RemoteAction action, RemoteActionResult re
       throw e;
     }
 
+    ImmutableList<ListenableFuture<FileMetadata>> forcedDownloads = forcedDownloadsBuilder.build();
```

Comment:
Remove this after making the changes below to unify `downloads` and `forcedDownloads`.

Reply:
See below

Reply:
See below